src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.ResponseDrain.cs

```diff
@@ -198,6 +198,7 @@ protected static string GetResponseForContentMode(string content, ContentMode mo
 
         // Similar to above, these are semi-extreme cases where the response should never drain for any handler.
 
+        [ActiveIssue(28423)]
```

Comment:
It's failing/hanging with all handlers?

Reply:
I thought it was both WinHttpHandler and SocketsHttpHandler.  But further testing shows just WinHttpHandler having issues.  

I will revise the disabling of the test.